.circleci/http_cache_test.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+set -eu
+
+trap 'killall http_cache' SIGINT SIGTERM EXIT
+
+DIR="${1:-/tmp/please}"
+
+# Extract the plz installation from earlier step
+rm -rf "$DIR"
+mkdir "$DIR"
+tar -xzf /tmp/workspace/linux_amd64/please_*.tar.gz --strip-components=1 -C "$DIR"
+ln -s "${DIR}/please" "${DIR}/plz"
+export PATH="$DIR:$PATH"
+
+# Start the server in the background
+plz run parallel -p -v notice --colour --detach //tools/http_cache:run_local
+
+# Test we can rebuild plz itself.
+plz test --profile localcache -p -v notice --colour //src/...
+
+# Run that again to make sure cache restoration works
```

Comment:
Do you want a clean or smthn in between?

Reply:
Ah that's a point. Clean doesn't clean the http cache so this should help actually use it.